lawnchair/src/app/lawnchair/search/SearchTargetUtils.kt

```diff
@@ -7,14 +7,23 @@ import androidx.core.os.bundleOf
 import com.android.app.search.LayoutType
 import com.android.launcher3.model.data.AppInfo
 import com.android.launcher3.util.ComponentKey
+import java.security.MessageDigest
+
+// We're generate hash key as alt in id, so we can avoid
+// the wrong action when click,
+// TODO remove when we properly manage creating search target
+private fun generateHashKey(input: String): String =
+    MessageDigest.getInstance("SHA-256")
+        .digest(input.toByteArray())
+        .joinToString("") { "%02x".format(it) }
```

Comment:
```suggestion
        .joinToString() { "%02x".format(it) }
```

Reply:
`""` is unnecessary?

Reply:
Oh yeah, I forgot to remove that since I use that to joined key like "shortcut_"